scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -48,12 +52,111 @@ public static BuildTargetResult Publish(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
```

Comment:
Might be missing it, but how does this get executed, what target has a dependency on it?